apps/src/sites/studio/pages/sections/new.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
I think 'new' is totally fine, but would maybe expect something like 'create'?

Reply:
I was following the rails convention where `new` is the page and `create` is the post call to create the sections. I feel like these routes might end up changing in the future but it seemed useful to follow those conventions for now

Reply:
🙌🏼